src/users/users.service.ts

```diff
@@ -269,7 +271,7 @@ export class UsersService {
             ) AS rank
           FROM
             users
-          JOIN
+          INNER JOIN
```

Comment:
If I inner join with `user_latest_events`, will the search show a user if they don't have points? That'd seem a little confusing to me if I want to look at my profile but haven't participated yet.

Reply:
Yup you are right, users would be filtered out here if they have no points. So should we only filter them from list result?